lib/sailor.js

```diff
@@ -202,23 +201,23 @@ class Sailor {
         self.messagesCount += 1;
 
         const timeStart = Date.now();
-        const traceId = message.properties.headers[AMQP_HEADER_TRACE_ID] || 'unknown';
+        const threadId = message.properties.headers.threadId || 'unknown';
         const messageId = message.properties.headers.messageId || 'unknown';
         const parentMessageId = message.properties.headers.parentMessageId || 'unknown';
         const deliveryTag = message.fields.deliveryTag;
-        log.info({
+        log.trace({
```

Comment:
is it possible to create logger.child to avoid copy&pasting during logger calls?